### PR TITLE
Feat: Add design tag to Letter template

### PR DESCRIPTION
**Pull request by abeddow91**
***

# WAITING ON FINAL EDITORIAL APPROVAL : DO NOT MERGE 

***

## What does this change?
Add a design tag (or "lozenge") to the DCR letter design that clicks through to the Letters tone page. It also adds a story.

Separates Letter into its own layout in AR and add a design tag.

Design is aware that "Letters" is also in the Byline. Instead of adjusting this in code or retroactively fixing this in CAPI, Editorial will fix forward by not writing this in future Letter articles. 

## Why?
This is a design decision from the Editorial Design team.

## Screenshots - DCR

| Before      | After      |
|-------------|------------|
| ![before-mobile][] | ![after-mobile][] |

[before-mobile]: https://user-images.githubusercontent.com/20416599/201634100-63b59e8e-04c4-42fc-a7f1-9882569f1af7.png
[after-mobile]: https://user-images.githubusercontent.com/20416599/201638135-e05207f7-e912-4b6b-bb1f-08f23e802157.png

| Before      | After      |
|-------------|------------|
| ![before][] | ![after][] |

[before]: https://user-images.githubusercontent.com/20416599/201633854-378a730f-76c0-40ae-9fc0-2fab51ab27fa.png
[after]: https://user-images.githubusercontent.com/20416599/201638110-4f69b793-a68d-4f36-9e48-4a65fae400ac.png

## Screenshots - AR

| Before      | After      |
|-------------|------------|
| ![before-ar][] | ![after-ar][] |

[before-ar]: https://user-images.githubusercontent.com/20416599/201659537-1b07bd91-48ad-4d3d-af7e-d67f40aa2a6e.png
[after-ar]: https://user-images.githubusercontent.com/20416599/201885639-11fbd786-f153-409a-bdd2-e8abceef12a1.png

### CODE REVIEW

**Review by mxdvl — APPROVED** [review]
Nice

**Review by tkgnm — APPROVED** [review]

**Review by JamieB-gu — APPROVED** [review]

**Review by JamieB-gu — APPROVED** [review]

File: dotcom-rendering/src/web/components/DesignTag.stories.tsx [hunk 1]
```diff
@@ -44,6 +44,44 @@ export const Interview = () => {
...
+				format={{
+					design: ArticleDesign.Letter,
+					display: ArticleDisplay.Standard,
+					theme: ArticlePillar.Sport,
+				}}
+			/>
+		</div>
+	);
+};
+Interview.story = { name: 'with design Letter' };
```
Comment (abeddow91) [inline dotcom-rendering/src/web/components/DesignTag.stories.tsx:83]:
```suggestion
Letter.story = { name: 'with design Letter' };
```

**Review by georgeblahblah — APPROVED** [review]
Looks good!

I think we _could_ continue with the Comment layout unless I missed something, but this approach is good too!

**Review by JamieB-gu — APPROVED** [review]

**Comment by abeddow91** [comment]
> Looks good!
> 
> I think we _could_ continue with the Comment layout unless I missed something, but this approach is good too!

Yeah I was wondering this but I wasn't sure to what extent we wanted nested `design.` template logic within a `design.` template and it seemed else where we had broken into a new layout. I suppose making the name of comment layout more generic might help.

**Pull request merged**